src/browserbehavior/ExtendedBrowserBehavior.ts

```diff
@@ -18,4 +18,5 @@ export default interface ExtendedBrowserBehavior extends BrowserBehavior {
   requiresContextRecreationForAudioWorklet(): boolean;
   isSimulcastSupported(): boolean;
   supportsBackgroundFilter(): boolean;
+  supportsResolutionScaling(): boolean;
```

Comment:
feels a bit unnecessarily unspecific.  Also 'support' feels like the wrong word.  `shouldDisableScaleDownBy`? or something?

Reply:
change to `disableScaleResolutionDownBy`